rocAL/rocAL/include/loaders/image/image_loader.h

```diff
@@ -38,7 +38,7 @@ class ImageLoader : public LoaderModule {
     ~ImageLoader() override;
     LoaderModuleStatus load_next() override;
     void initialize(ReaderConfig reader_cfg, DecoderConfig decoder_cfg, RocalMemType mem_type, unsigned batch_size, bool keep_orig_size=false) override;
-    void set_output_image (Image* output_image) override;
+    void set_output (Tensor* output_tensor) override;
```

Comment:
Please remove the space after set_output. Applicable to all such instances of this function

Reply:
Done